gramineproject/gramine
# [Docs] Move "Management Team" to a separate file

## Description of the changes 

This is in preparation for the bigger reorganization of documentation.

Extracted from https://github.com/gramineproject/gramine/pull/1241.

## How to test this PR? 

I created a RTD job to see the rendering: https://gramine.readthedocs.io/en/dimakuv-docs-mv-management-team/management-team.html#the-procedure-for-adding-and-removing-maintainers. Notice the "Management Team" menu. Compare with our classic: https://gramine.readthedocs.io/


- - -
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/gramineproject/gramine/1347)
